index.html

```diff
@@ -140,7 +140,35 @@ <h1 class="title_post">Multi-Post Stories</h1>
   
       <section class="button">See Project</section>
     </section>
-
+    <section id="contact-me">
+      <header>
```

Comment:
- [ ] Please you should not use the `header ` tag in the other section beside the navbar section, to make your code more semantically correct I would recommend you remove it.